essential-REST.md

```diff
@@ -0,0 +1,56 @@
+After listening to shows #185 and #189, both on APIs, I started realizing that perhaps there still isn't one central resource for learning about REST APIs. So, I've put together this essential reading list for those of you looking for some of the best resources out there on RESTful APIs.
+
+**[Objects at REST, Part 1: What Is It?](https://medium.com/@Stuart_Weir/objects-at-rest-part-1-what-is-it-3e26f0978616#.rr8x0ahz0)**
+
+This was written recently by me, describing (in one post) many of the fundamental principles, terminology, and ideas behind RESTful APIs. While it's not too granular, I goes into some detail about what REST APIs really are to build your understanding of the technology quickly.
+> An API is an Application Programming Interface. This sounds super technical, but it actually really isn’t. It’s essentially an interface for programmers, much like a User Interface (UI) is an interface for product users.
+
+**[REST API Tutorial](http://www.restapitutorial.com/)**
+
+An excellent resource on REST APIs, REST API Tutorial offers a number of tutorials on REST, quick tips, HTTP method use,resource naming, and a number of other REST API best practices. A _must read_.
```

Comment:
I think we can scratch:

> An excellent resource on REST APIs,

The fact that it made your list means you already believe this.